### PR TITLE
Automatically remove old plugin scores

### LINKED ISSUE #193: Automatically removing old scores for each plugin
We can keep 5 or 10 scores for each plugins to get a trend, but we don't need to keep all the scores ever computed.
This needs to be scheduled and well tested.

- [ ] Removes scores for each plugins except the last 5 (by `computed_at`).
- [ ] Schedule this process to run every hour.
- [ ] This needs to be tested.

### PR DESCRIPTION
**Pull request by Sayo-gs**
resolved #193

### CODE REVIEW

**Comment by Sayo-gs** [comment]
can you tell me why that 1 check is not passed

**Comment by Sayo-gs** [comment]
Hey @alecharp 
can you guide me why that single check was not passed

File: core/src/main/java/io/jenkins/pluginhealth/scoring/service/ScoreService.java [hunk 1]
```diff
@@ -68,6 +68,25 @@ public Map<String, ScoreSummary> getLatestScoresSummaryMap() {
...
+
+              repository.deleteAll(scoresToRemove);
+            }
+        }
+    }
+
+     @Scheduled(cron = "0 0 * * * *")
+    public void scheduledDeleteOldPluginScores() {
+        deleteOldPluginScores();
+    }
```
Comment (alecharp) [inline core/src/main/java/io/jenkins/pluginhealth/scoring/service/ScoreService.java:88]: I'd prefer this to be on a separate class, most probably in `war` module.

**Review by alecharp — CHANGES_REQUESTED** [review]
The pull request template is here to guide you, as well as the contributing guide. It would have helped you. 

Please see https://github.com/jenkins-infra/plugin-health-scoring/pull/271/checks?check_run_id=12455363353

**Comment by alecharp** [comment]
Because of https://github.com/jenkins-infra/plugin-health-scoring/pull/271/checks?check_run_id=12455363353

File: core/src/main/java/io/jenkins/pluginhealth/scoring/service/ScoreService.java (hunk 1 above)
Comment (GauravFin) [reply core/src/main/java/io/jenkins/pluginhealth/scoring/service/ScoreService.java:88]: where should it be placed in between schedule or service folder in war module, most probably schedule folder

Comment (alecharp) [reply core/src/main/java/io/jenkins/pluginhealth/scoring/service/ScoreService.java:88]: it can be in the scoring scheduler yes.

**Pull request closed without merging**